meson_options.txt

```diff
@@ -2,6 +2,6 @@ option('bundle', type : 'boolean', value : false, description: 'Build a macOS bu
 option('source-only', type : 'boolean', value : false, description: 'Configure source files only, doesn\'t checks for dependencies')
 option('portable', type : 'boolean', value : false, description: 'Portable install')
 option('renderer', type : 'boolean', value : false, description: 'Use SDL renderer')
-option('dirmonitor_backend', type : 'combo', value : '', choices : ['', 'inotify', 'fsevents', 'kqueue', 'win32', 'dummy'], description: 'define what dirmonitor backend to use')
+option('dirmonitor_backends', type : 'array', value : [], choices : ['', 'inotify', 'fsevents', 'kqueue', 'inodewatcher', 'win32', 'dummy'], description: 'define what dirmonitor backend to use')
```

Comment:
since list can be empty the empty string should be removed

Reply:
yes.